Prevent calling Git to obtain date of the last commit, and hardcode
[last_git_commit] value to match package version.

Index: configure.ac
--- configure.ac.orig
+++ configure.ac
@@ -23,9 +23,9 @@ dnl Foundation, Inc., 59 Temple Place, Suite 330, Bost
 
 dnl PACKAGE_VERSION will be either "" if not using git, or date of the last git commit in the form YYYYMMDD
 m4_define([last_git_commit], patsubst(m4_esyscmd([git log -n 1 --date=short --format=format:%cd]), [[^0-9]]))
-AC_INIT([schismtracker], [last_git_commit])
+AC_INIT([schismtracker], [20240630])
 
-last_commit_date=[last_git_commit]
+last_commit_date=[20240630]
 
 AC_CONFIG_SRCDIR([schism/main.c])
 
